lib/ckb/multi_sign_wallet.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+
+module CKB
+  class MultiSignConfiguration
+    attr_reader :require_n
+    attr_reader :threshold
+    attr_reader :pubkeys
+
+    def initialize(require_n:, threshold:, pubkeys:)
+      raise "require_n should be less than 256" if require_n > 255 || require_n < 0
+      raise "threshold should be less than 256" if threshold > 255 || threshold < 0
+      raise "Pubkey number must be less than 256" if pubkeys.length > 255
+
+      @require_n = require_n
+      @threshold = threshold
+      @pubkeys = pubkeys
+    end
+
+    def serialize
+      CKB::Utils.bin_to_hex(
+        [0, require_n, threshold, pubkeys.length].pack("CCCC") +
+        pubkeys.map do |pubkey|
+          CKB::Utils.hex_to_bin(pubkey)
```

Comment:
Just a note: due to [this change](https://github.com/nervosnetwork/ckb/pulls), pubkey here needs another blake160 hash.